### PR TITLE
Update mpd.rst

### PR DESCRIPTION
**Pull request by piroxxi**
Add Mopidy Wombat Client to the list of android clients for Mopidy

### CODE REVIEW

File: docs/clients/mpd.rst [hunk 1]
```diff
@@ -264,6 +264,22 @@ This app looks terrible in the screen shots, got just 100+ downloads, and got a
...
+Test date:
+    2015-04-03
+Tested version:
+    1.1 (released 2015-04-03)
+
+You can get `Mopidy Wombat Client
+<https://play.google.com/store/apps/details?id=fr.piroxxi.mopidy.wombatclient>`_.
+
+Mopidy Wombat Client is designed to work well with Mopidy plugged to Spotify. The
+application shows a tree of your playlists and let you play them with a simple
```
Comment (kingosticks) [inline docs/clients/mpd.rst]: s/let/lets/

**Comment by kingosticks** [comment]
Did you run through the test actions defined above? Be nice to say that it did or didn't pass them all.

**Comment by kingosticks** [comment]
Oops! Sorry

**Comment by jodal** [comment]
Since this PR was submitted, we've cleaned up the MPD client list a lot, removing most old cruft. For now, I'm tempted to let the MPD client listing stay quite static as a list of "these clients can be recommended" from the 100+ clients available, as opposed to the Mopidy web client listing, which tries to be quite complete. Longer term, I'm working on a registry site for Mopidy extensions/clients/related projects. I think this fits better there.

**Pull request closed without merging**